src/networking.c

```diff
@@ -4224,6 +4224,9 @@ void helloCommand(client *c) {
 
     addReplyBulkCString(c, "modules");
     addReplyLoadedModules(c);
+
+    addReplyBulkCString(c, "availability_zone");
+    addReplyBulkCString(c, server.availability_zone);
```

Comment:
Shall we only include it if it's non-empty? So it doesn't take any space when availability zone is not used. I think we should keep the HELLO response small.

Reply:
updated.